Description: Disable three touch tap in favor of uTouch gestures.
Bug-Ubuntu: https://launchpad.net/bugs/961725

Index: xserver-xorg-input-synaptics-1.8.2/src/synaptics.c
===================================================================
--- xserver-xorg-input-synaptics-1.8.2.orig/src/synaptics.c
+++ xserver-xorg-input-synaptics-1.8.2/src/synaptics.c
@@ -625,7 +625,7 @@ set_default_parameters(InputInfoPtr pInf
     /* Enable tap */
     tapButton1 = 1;
     tapButton2 = 3;
-    tapButton3 = 2;
+    tapButton3 = 0; /* Disabled by default so three-touch gestures work */
 
     /* Enable multifinger-click if only have one physical button,
        otherwise clickFinger is always button 1. */
